libs/ledger-live-common/src/naming-service/logic.ts

```diff
@@ -11,13 +12,12 @@ export function isOutdated(resource: NamingServiceStatus): boolean {
       return now - resource.updatedAt > 60 * 1000; // 1 minute
     }
     case "error": {
-      return now - resource.updatedAt > 1 * 1000; // 1 second
+      return now - resource.updatedAt > 30 * 1000; // 30 seconds
     }
   }
   return false;
 }
 
 export const isNameValid = (name: string | undefined): boolean => {
-  const regex = /[.*]\w{1,}/g;
-  return !!(name && regex.test(name));
+  return VALID_DOMAINS.some(str => name?.includes(str))
```

Comment:
```suggestion
  return VALID_DOMAINS.some(str => name?.endsWith(str))
```

It's unlikely to happen but this function would return true if we have `foo.eth.xyz` or `foo.ethx`. not sure if we want it to be a valid case